service/resource/deployment/deployment.go

```diff
@@ -0,0 +1,44 @@
+package deployment
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/azuretpr"
+
+	"github.com/giantswarm/azure-operator/service/key"
+)
+
+const (
+	clusterIDParameterName     = "clusterID"
+	clusterSetupDeploymentName = "cluster-setup"
```

Comment:
For now we use the cluster-setup template. This will be replaced by the master template.